spanner_orm/condition.py

```diff
@@ -37,6 +37,9 @@ def __init__(self):
     self.model = None
     self.suffix = None
 
+  def __eq__(self, obj):
+    return isinstance(obj, self.__class__) and self.model == obj.model
```

Comment:
use ```type(self)``` instead of ```self.__class__```
though no children use this by itself, so i'm not sure we should actually put this in the base class